ui/src/components/Navbar/Navbar.scss

```diff
@@ -51,7 +52,7 @@
   }
 
   &__left-group {
-    padding-right: $aleph-grid-size;
+    @include rtlSupportInvertedProp(padding, right, $aleph-grid-size, inherit);
```

Comment:
For the LTR value I think this should be 0, not inherit.  Right now it's adding extra left padding in the LTR view